starfish/pipeline/filter/bandpass.py

```diff
@@ -68,4 +68,5 @@ def filter(self, stack) -> None:
 
         # apply to aux dict too:
         for k, val in stack.aux_dict.items():
-            stack.aux_dict[k] = self.bandpass(val, self.lshort, self.llong, self.threshold, self.truncate)
+            stack.aux_dict[k].numpy_array = self.bandpass(
```

Comment:
Ah cool, these filters can all be way simpler now. Since the entries in `aux_dict` are all `ImageStack` objects all you need to do is: 
```python
for image_stack in stack.aux_dict.values():
    image_stack.apply(bandpass_)
```